src/App.js

```diff
@@ -10,20 +11,17 @@ import Navigation from "./components/Navigation";
 import Div100vh from "react-div-100vh";
 
 function App() {
-  const [menuItems, setMenuItems] = useState([]);
+  const [menu, setMenu] = useState([]);
+  const [order, setOrder] = usePersistedState("order", []);
 
-  const history = useHistory();
+  let history = useHistory();
 
   useEffect(() => {
```

Comment:
this could also be a hook or a helper function and instead of an empty array, you pass the function as InitialValue into your state in line 14.